content/block-3/index.md

```diff
@@ -1,56 +1,42 @@
 [« Back to Index](../../README.md)
 
-# Block 3: Start sample project (2 weeks)
+# Block 3: Design (2 weeks)
 
 ## Contents
-
-- GraphQL.
-- ORM (Sequelize).
+- Component Diagrams
+- Advanced CSS
```

Comment:
🤔 the focus of this block should be only `Component Diagrams` and `Team Collaboration`. CSS and FE Architecture is Block 4.